services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -724,6 +811,10 @@ def _update_workbench(
                     .returning(literal_column("*"))
                 )
                 project: RowProxy = await result.fetchone()
+
+                for node_update_task in nodes_update_tasks:
+                    fire_and_forget_task(node_update_task)
```

Comment:
did you check what happens if you have computational services running and stuff like that? you do not end up in a loop?

Reply:
yes, I've checked, and sorry for the bad naming. This is only for frontend services. And for now of all the frontend services only the file picker will be used. I've changed the names to reflect this.